File: skills/quality/dry-test/fixtures/cpp/clone.cpp
#include <string>

std::string alpha(int value) {
    // Only alpha carries a comment. Comments are spelling, so beta must still match it.
    std::string label = "alpha: ";
    int total = value + 1;
    if (total > 10) {
        return label + std::to_string(total);
    }
    return label;
}

std::string beta(int number) {
    std::string tag = "beta: ";
    int amount = number + 2;
    if (amount > 20) {
        return tag + std::to_string(amount);
    }
    return tag;
}

File: skills/quality/dry-test/fixtures/python/clone.py
def alpha(value):
    # Only alpha carries a comment. Comments are spelling, so beta must still match it.
    label = "alpha: "
    total = value + 1
    if total > 10:
        return label + str(total)
    return label


def beta(number):
    tag = "beta: "
    amount = number + 2
    if amount > 20:
        return tag + str(amount)
    return tag
